### DISCUSSION TITLE
Any further updates?

### DISCUSSION THREAD

**Discussion by PokeGunk** [danenders/pokeclassic#149] [General]
Was looking forward to playing this as it's the probably the best hack I've ever seen, was hoping for glitches to be fixed for a final release.

**Comment by danenders** [comment]
Work & personal life has gotten in the way but users here & on the old Reddit post have mentioned fixing some of the remaining glitches on their own. 

I will come back around compile it sometime but no ETA.

**Reply by vfasousa10** [reply]
Rock smash is not listed under HMs nor TMs; is there a pokemon that learns it naturally? I can't complete the game without it :/

**Reply by TheBenjamoose** [reply]
Upstairs in the Pokémon centre just before the mt moon entrance there is a hiker. Chat to him to get rock smash.

if anyone knows a location for a 4th moonstone that would be handy. Or do I hunt clefairy hoping they are holding one?
